lib/Doctrine/Common/Reflection/Psr0FindFile.php

```diff
@@ -35,11 +35,12 @@ public function __construct($prefixes)
      */
     public function findFile($class)
     {
-        $lastNsPos = strrpos($class, '\\');
         if ($class[0] === '\\') {
             $class = substr($class, 1);
         }
 
+        $lastNsPos = strrpos($class, '\\');
```

Comment:
Found this bug when I was writing a test for the if condition above. Since we modify the `$class` variable with substr, the strrpos value is wrong. It isn't anything we use or rely on but figured I would fix it.